.circleci/config.yml

```
@@ -58,7 +58,7 @@ jobs:
       - setup
       - run: swift build
 
-  test-swift-5-0:
+  test-swift-5-2:
```

Since this breaks the required CI status checks, maybe it'd be easier to keep as-is, since "5.0" really means "5.2", at least as far as `SWIFT_VERSION` is concerned

```suggestion
  test-swift-5-0:
```